message UnitMoveMsg
{
    required int32 unitid = 1;
    required uint32 position = 2;
}

message ChangeMsg
{
    optional UnitMoveMsg unitmove = 1;
}

